sqlmesh/cli/main.py

```diff
@@ -164,7 +166,10 @@ def render(
     )
 
     sql = rendered.sql(pretty=True, dialect=ctx.obj.config.dialect if dialect is None else dialect)
-    ctx.obj.console.show_sql(sql)
+    if no_format:
+        print(sql)
+    else:
+        ctx.obj.console.show_sql(sql)
```

Comment:
I think this needs to instead be `pretty=not no_format` or something?

Reply:
what do you mean?

Reply:
i may have misunderstood the problem - is it that we're pretty-printing, or that we're using the fancy rich printer instead of python's `print`?

Reply:
fancy rich printer, that confused me too

Reply:
oh i just re-read the description, it seems it's the latter  🤦 -- ignore me

Reply:
The intent is actually just printing rendered query to stdout **without** going through `rich.print` + rich console + `rich.syntax` which the combination were adding unnecessary noise to stdout, which made the command difficult to use. I think `--pretty/no-pretty` could be a separate option -- fyi @GeorgeSittas

Reply:
Gotcha thanks

Reply:
Re: I think --pretty/no-pretty could be a separate option

Sure, that could be helpful I guess if you wanna avoid SQLGlot's formatter for any reason? cc @tobymao

Reply:
can be done later